### DISCUSSION TITLE
TypeScript raises error when I use `useScrollIntoView` hook

### DISCUSSION THREAD

**Discussion by PhanDungTri** [mantinedev/mantine#615] [Q&A] [answered]
Hello everyone,

When I assign `targetRef` to `ref` prop of the component, TypeScript raises this error at `ref` property.

![image](https://user-images.githubusercontent.com/26342804/147897738-23062bc4-23c9-45b7-a16e-ec406e88d76c.png)

And here is how I use it. I just follow the documentation.

![image](https://user-images.githubusercontent.com/26342804/147897798-f79abca3-ff03-4705-b618-f7288eaa6822.png)

I don't know if I'm doing something wrong or this is a issue of Mantine. Any idea? Thank you!

**Environment:**
- Ubuntu 20.04.3 LTS
- Matine 3.4.1
- TypeScript 4.1.2

**Comment by rtivital** [comment, marked as answer]
You can set hook type, see definition – https://mantine.dev/hooks/use-scroll-into-view/#definition
```tsx
useScrollIntoView<HTMLDivElement>();
```

**Reply by PhanDungTri** [reply]
Thank you.
